sfe.bll/UserLogic.cs

```diff
@@ -9,65 +9,65 @@
 
 namespace sfe.bll
 {
-    public class AgentLogic
+    public class UserLogic
     {
         private static DataClassesDataContext db = Database.Instance;
-        public static List<Agent> Get()
+        public static List<User> Get()
         {
             try
             {
-                return (from agents in db.Agents
-                        where agents.active == true
-                        select agents).ToList();
+                return (from users in db.Users
+                        where users.active == true
+                        select users).ToList();
             }
             catch (Exception e)
             {
                 EventLog.WriteEntry(e.Source, e.Message);
```

Comment:
Ok, please verify in the event viewer if that information is that you required to solve possible issues. Add a severity level (info, warn, error)